app/api/v2/admin/users.rb

```diff
@@ -225,6 +225,24 @@ def search(field, value)
               label.destroy
               status 200
             end
+
+            desc 'Returns array of users as paginated collection',
+            security: [{ "BearerToken": [] }],
+            failure: [
+              { code: 401, message: 'Invalid bearer token' }
+            ]
+            params do
+              requires :key,      type: String, desc: 'Label key'
+              requires :value,    type: String, desc: 'Label value'
+              optional :page,     type: Integer, default: 1,   integer_gt_zero: true, desc: 'Page number (defaults to 1).'
+              optional :limit,    type: Integer, default: 100, range: 1..1000, desc: 'Number of users per page (defaults to 100, maximum is 1000).'
+            end
+            get '/search' do
+              users = User.joins(:labels).where(labels: { scope: 'private', key: params[:key], value: params[:value] })
```

Comment:
I think we need to remove private scope from the search, label can be public
I think we can improve this api

Reply:
then, should we add a new optional parameter to specify?
Cause mixing private and public is not so good idea

Reply:
ok, sounds good but maybe /search is not the right name or maybe we could do more search possibilities ?